Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -490,8 +590,25 @@ def inclusion {n : ℕ} : SimplexCategory.Truncated n ⥤ SimplexCategory :=
 instance (n : ℕ) : (inclusion : Truncated n ⥤ _).Full := FullSubcategory.full _
 instance (n : ℕ) : (inclusion : Truncated n ⥤ _).Faithful := FullSubcategory.faithful _
 
+/-- A proof that the full subcategory inclusion is fully faithful.-/
+noncomputable def inclusion.fullyFaithful (n : ℕ) :
+    (inclusion : Truncated n ⥤ _).op.FullyFaithful := Functor.FullyFaithful.ofFullyFaithful _
+
+@[ext]
+theorem Hom.ext {n} {a b : Truncated n} (f g : a ⟶ b) :
+    f.toOrderHom = g.toOrderHom → f = g := SimplexCategory.Hom.ext _ _
+
 end Truncated
 
+/-- An abbreviation for the full subcategory of the simplex category spanned by the objects
+0, ..., k. -/
+abbrev Δ (k : ℕ) := SimplexCategory.Truncated k
```

Comment:
I would very much prefer if these abbreviations were not introduced because I find them extremely confusing.

Reply:
Cut.

Reply:
Done.